servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -60,6 +60,8 @@ nessie.server.send-stacktrace-to-client=false
 #   && role=='admin_user'
 # nessie.server.authorization.rules.allow_listing_reflog=\
 #   op=='VIEW_REFLOG' && role=='admin_user'
+# nessie.server.authorization.rules.allow_deleting_default_branch=\
+#   op=='DELETE_DEFAULT_BRANCH' && role=='admin_user'
```

Comment:
imo its a bit odd that `VIEW_REFLOG` is not part of `allow_all` rule... but i followed this "new pattern" here.

Reply:
Generally, the CEL expressions and rules are flexible and users can create/use what they need/want.

The examples are probably a bit misleading.

The VIEW_REFLOG example should probably use a different role name (superuser? gc-user?) and the example for DELETE_DEFAULT_BRANCH maybe use the "superuser" role. IDK.
But honestly, the "test_user" role names above are similar.